addons/rose/app/styles/rose/components/header/_dropdown.scss

```diff
@@ -3,35 +3,10 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-@use '../../variables/sizing';
-@use '../../utilities/type';
-@use '../../variables/color/sass';
-
-$chevron: "data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%23bac1cc' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z'></path></svg>";
-
 .rose-header {
-  .rose-dropdown {
-    &,
-    &[open] {
-      .rose-dropdown-trigger {
-        color: #{sass.color(ui-gray, 300)};
-        background: url($chevron) no-repeat right sizing.rems(s) center;
-        background-size: sizing.rems(m);
-
-        $y-padding: sizing.rems(xxxs);
-        $right-padding: sizing.rems(xs) + sizing.rems(l);
-
-        padding: $y-padding $right-padding $y-padding sizing.rems(s);
-
-        &.has-icon-only {
-          padding-left: sizing.rems(xs);
-        }
-
-        &:hover,
-        &:focus {
-          background-color: transparent;
-        }
-      }
-    }
+   .hds-dropdown > button {
+    background: var(--black);
+    border: none;
+    color: var(--ui-gray-subtler-3);
```

Comment:
Is the contrast a little low in dark mode with this? Should it just be `--white`?

<img width="129" alt="image" src="https://github.com/user-attachments/assets/88694202-300b-4d63-a907-f0f2b8dcfd45" />